pkgs/data/fonts/vista-fonts/default.nix

```diff
@@ -21,7 +29,9 @@ fetchzip {
     done
   '';
 
-  sha256 = "sha256-x7JSXS9Q1fzlJTVR+MAS3f2+cmo/H0s1qkY9FPjx2zI=";
+  outputHashAlgo = "sha256";
+  outputHashMode = "recursive";
+  outputHash = "0cnvy7w18ga6m8sln7rzd9rbxzfx2b0ghl9m4pjzrmah5xfm5cn7";
```

Comment:
mkDerivation does not require this.

Reply:
It makes the hash depend only on the result, so it avoids unnecessary rebuilds. The package contains font files that won't change if the environment is updated. Other font packages, such as corefonts or cantarell-fonts also do this.

Reply:
How long does the package take to build? If it only takes 5 seconds than this is not worth it.

Reply:
Removed then